### PR TITLE
yes

### PR DESCRIPTION
**Pull request by EggY0lk**
absolutely

### CODE REVIEW

File: src/main/java/RedBot/Bot.java [hunk 1]
```diff
@@ -87,12 +87,20 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
...
-                    "Very doubtful"
+                    "Very doubtful", 
+                    //shit after here is added by Egg
+                    "Are you sure you want to ask that?",
+                    "Perhaps",
+                    "Sure, why not eh",
+                    "Absolutely not you donkey!",
+                    "Stop asking me dumb questions"
+                    "Wow, that is actually a good question, FINALLY!"
+                    "Tbh, I have no idea."
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:98]: Some of these aren't even a clear yes or no answer

File: src/main/java/RedBot/Bot.java [hunk 2]
```diff
@@ -87,12 +87,20 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
...
                 };
                 event.getChannel().sendMessage(":8ball: "+answer[r.nextInt(answer.length-1)]).queue();
                 break;
             
-            case "pick": //Chooses an option from a selection
+            case "pick": //Chooses an option from a selection      me when finding the right hentai
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:103]: Bruh

File: src/main/java/RedBot/Bot.java [hunk 3]
```diff
@@ -119,7 +127,7 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
                         String[] choices = fullstring.split(", ");
                         if(choices.length==1)
                             choices = choices[0].split(",");
-                        /* Duplicate check: Doesn't work for some reason
+                        /* Duplicate check: Doesn't work for some reason    :pensive:
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:130]: Emotes don't work in code

File: src/main/java/RedBot/Bot.java [hunk 4]
```diff
@@ -138,7 +146,7 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
                 }
                 break;
                 
-            case "hentai": // TODO Optimise this train wreck
+            case "hentai": // TODO Optimise this train wreck            Good fucking luck lmao -Egg
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:149]: I'd appreciate it if you actually tried making it work

File: src/main/java/RedBot/Bot.java [hunk 5]
```diff
@@ -173,7 +181,7 @@ else if(!event.getTextChannel().isNSFW()){
                         if(i!=rawTags.size()-1)
                             tags += ", ";
                     }
-                    Elements thumbnail = Page.getData("thumb"); //Get the thumbnail
+                    Elements thumbnail = Page.getData("thumb"); //Get the thumbnail ( ͡° ͜ʖ ͡°)
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:184]: Bruh 3

File: src/main/java/RedBot/Bot.java [hunk 6]
```diff
@@ -186,7 +194,7 @@ else if(!event.getTextChannel().isNSFW()){
                 
             case "mc":
             case "minecraft":
-                try { // Shows info on the Minecraft server
+                try { // Shows info on the Minecraft server         rip -Egg
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:197]: Bruh 4

File: src/main/java/RedBot/Bot.java [hunk 7]
```diff
@@ -276,5 +284,6 @@ public String help(){
                 + "\n"
                 + "`d.minecraft`:\n"
                 + "Shows info and/or status of the Minecraft server";
-    }
+                + "or just ping MiloAddict";
+    }  //You expected a comment but it was me DIO!
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Bot.java:288]: Bruh 5

File: src/main/java/RedBot/Bot.java (hunk 1 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:98]: its 8ball.

File: src/main/java/RedBot/Bot.java (hunk 2 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:103]: ngl, idk wtf happened there, got horny maybe

File: src/main/java/RedBot/Bot.java (hunk 3 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:130]: its a comment

File: src/main/java/RedBot/Bot.java (hunk 4 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:149]: idk how to 😔

File: src/main/java/RedBot/Bot.java (hunk 5 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:184]: ( ͡° ͜ʖ ͡°)

File: src/main/java/RedBot/Bot.java (hunk 6 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:197]: dorime

File: src/main/java/RedBot/Bot.java (hunk 7 above)
Comment (EggY0lk) [reply src/main/java/RedBot/Bot.java:288]: not wrong

File: src/main/java/RedBot/Bot.java (hunk 1 above)
Comment (LukeHuckman) [reply src/main/java/RedBot/Bot.java:98]: In any case, I followed the [official 8ball responses](http://www.otcpas.com/advisor-blog/magic-8-ball-messages/), so no need to add more.

**Pull request closed without merging**